From 5c9e8a994fe8529388ab159e0aa371c75bc3d17b Mon Sep 17 00:00:00 2001
From: MilhouseVH <milhouseVH.github@nmacleod.com>
Date: Thu, 21 Sep 2017 04:22:12 +0100
Subject: [PATCH] drm/pci: drm_pci_init/exit now deprecated

As per: https://github.com/torvalds/linux/commit/10631d724deff712343d96dd3017cd323349f761

Use legacy variant, as pci_[un]register_driver currently fails.
---
 kernel/nv-drm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/kernel/nv-drm.c b/kernel/nv-drm.c
index 2aa7cc8..23df996 100644
--- a/kernel/nv-drm.c
+++ b/kernel/nv-drm.c
@@ -173,7 +173,7 @@ int __init nv_drm_init(
 {
     int ret = 0;
 #if defined(NV_DRM_AVAILABLE)
-    ret = drm_pci_init(&nv_drm_driver, pci_driver);
+    ret = drm_legacy_pci_init(&nv_drm_driver, pci_driver);
 #endif
     return ret;
 }
@@ -183,7 +183,7 @@ void nv_drm_exit(
 )
 {
 #if defined(NV_DRM_AVAILABLE)
-    drm_pci_exit(&nv_drm_driver, pci_driver);
+    drm_legacy_pci_exit(&nv_drm_driver, pci_driver);
 #endif
 }
 
-- 
2.7.4

